yamale/command_line.py

```diff
@@ -112,8 +114,11 @@ def main():
     parser.add_argument('--strict', action='store_true',
                         help='Enable strict mode, unexpected elements in the data will not be accepted.')
     args = parser.parse_args()
-    _router(args.path, args.schema, args.cpu_num, args.parser, args.strict)
-    print('Validation success! 👍')
+    if _router(args.path, args.schema, args.cpu_num, args.parser, args.strict) == 0:
```

Comment:
Here, you could catch the exceptions and print out the result....then you don't need the 0/1 status.